src/overlays/actors/ovl_En_Stream/z_en_stream.c

```diff
@@ -29,19 +31,117 @@ const ActorInit En_Stream_InitVars = {
     (ActorFunc)EnStream_Update,
     (ActorFunc)EnStream_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Stream/func_80B0B7A0.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Stream/EnStream_Init.s")
+static InitChainEntry sInitChain[] = {
+    ICHAIN_VEC3F_DIV1000(scale, 20, ICHAIN_STOP),
+};
+
+void EnStream_SetupAction(EnStream* this, EnStreamActionFunc actionFunc) {
+    this->actionFunc = actionFunc;
+}
+
+void EnStream_Init(Actor* thisx, GlobalContext* globalCtx) {
+    EnStream* this = THIS;
+
+    this->unk_150 = thisx->params & 0xFF;
+    Actor_ProcessInitChain(thisx, sInitChain);
+    if ((this->unk_150 != 0) && (this->unk_150 == 1)) {
+        thisx->scale.y = 0.01f;
+    }
+    EnStream_SetupAction(this, &EnStream_WaitForPlayer);
```

Comment:
We omit the `&` for function and array pointers
```suggestion
    EnStream_SetupAction(this, EnStream_WaitForPlayer);
```